pom.xml

```diff
@@ -52,16 +52,17 @@
         <gravitee-cockpit-api.version>3.7.0</gravitee-cockpit-api.version>
         <gravitee-cloud-initializer.version>2.1.0</gravitee-cloud-initializer.version>
         <gravitee-spec-gen-api.version>1.1.0</gravitee-spec-gen-api.version>
-        <gravitee-common.version>4.5.1</gravitee-common.version>
+        <gravitee-common.version>4.6.0</gravitee-common.version>
+        <gravitee-secret-api.version>1.0.0-APIM-7497-api-secrets-SNAPSHOT</gravitee-secret-api.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**  <gravitee-secret-api.version>************************************</gravitee-secret-ap  
[This comment was created by Aqua Pipeline]